Sources/ResilientDecoding/ErrorReporting.swift

```diff
@@ -142,3 +185,32 @@ private extension Error {
 }
 
 #endif
+
+// MARK: - Private
+
+/**
+ In the unlikely event that `enableResilientDecodingErrorReporting()` is called multiple times, this error will be reported to the earlier `ResilientDecodingErrorReporter` to signify that the later one may have eaten some of its errors.
+ */
+private struct MayBeMissingReportedErrors: Error { }
```

Comment:
Do we really want this to be private?
We're including it in the errors we report to the user.
If it's private, how can they check for it?

Reply:
Its not really about checking for it, it is about having _something_ there if it happens (which it very likely won't)